### DISCUSSION TITLE
I demand the closure of the dissemination of content!

### DISCUSSION THREAD

**Discussion by 9564519** [PHPMailer/PHPMailer#2591] [General]
С помощью этого ПО было отправлено фишинговое письмо от моего имени, с стороннего почтового сервера! Я это письмо не отправлял. Следы привели именно к этому ПО с Гитхаба:
(GoogleTranslator) This software was used to send a phishing email on my behalf, from a third-party mail server! I did not send this letter. Traces led exactly to this software from GitHub.
https://ibb.co/PtsZYk5
(The report was framed as phishing - malware)

**Comment by Synchro** [comment]
Nope. They simply used this software (which by itself does nothing at all) within their own scripts, which we have no control over. Blame the sender, not the software, and learn how to secure your domain against forgeries and phishing.

**Comment by 9564519** [comment]
(I really hope I understood you correctly, and I really hope that the
google translator has not lost the meaning of the message, in any case,
thanks to google translator.)
No. The software was written by an attacker and made publicly available
through your GitHub servers. Any software doesn't do anything by itself.
Until the software is initiated by the user in the operating environment,
the computer. Following YOUR logic, there is no malware at all. Brilliant
logic. I will figure out whom to blame, I do not require your advice in
this area. In this situation, the software and its creator are to blame.
You will decide what is malicious and what is not. ! This program is
malicious. I showed you an example of her action. With its help, passwords
and other data of users on the Internet are stolen. It is extremely foolish
to argue with this. You are involved in the distribution of such a tool.

пт, 24 дек. 2021 г. в 21:45, Marcus Bointon ***@***.***>:

**Reply by Synchro** [reply]
No, the attacker's software is not available through GitHub. One of the entirely passive open-source libraries they used is. It's not my fault that you don't understand such simple concepts.
